01.md

```diff
@@ -85,10 +85,11 @@ As a convention, all single-letter (only english alphabet letters: a-z, A-Z) key
 
 ### Kinds
 
-Kinds specify how clients should interpret the meaning of each event and the other fields of each event (e.g. an `"r"` tag may have a meaning in an event of kind 1 and an entirely different meaning in an event of kind 10002). Each NIP may define the meaning of a set of kinds that weren't defined elsewhere. This NIP defines two basic kinds:
+Kinds specify how clients should interpret the meaning of each event and the other fields of each event (e.g. an `"r"` tag may have a meaning in an event of kind 1 and an entirely different meaning in an event of kind 10002). Each NIP may define the meaning of a set of kinds that weren't defined elsewhere. [NIP-10](10.md), for instance, especifies the `kind:1` text note for social media applications.  
+
+This NIP defines one basic kind:
 
 - `0`: **metadata**: the `content` is set to a stringified JSON object `{name: <username>, about: <string>, picture: <url, string>}` describing the user who created the event. A relay may delete older events once it gets a new one for the same pubkey.
```

Comment:
Why would it would be appropriate to move Kind 1 to a separate NIP, but not Kind 0?  The intention behind this PR would then seem to imply that Kind 0 is mandatory.

Reply:
Kind 0 is used by virtually all clients, not only Kind 1 clients. Especially since the original kind 0, only has support for 3 fields: `{name: <username>, about: <string>, picture: <url, string>}`